src/main/java/net/tropicraft/core/common/dimension/biome/TropicraftBiome.java

```diff
@@ -28,6 +32,10 @@ public void addFeatures() {
         // Volcano feature to add tile entity to the volcano generation. Checks in each chunk if a volcano is nearby.
         addFeature(GenerationStage.Decoration.SURFACE_STRUCTURES, createDecoratedFeature(TropicraftFeatures.VOLCANO.get(), IFeatureConfig.NO_FEATURE_CONFIG, Placement.NOPE, IPlacementConfig.NO_PLACEMENT_CONFIG));
 //
+
+        addStructure(TropicraftFeatures.HOME_TREE.get(), new VillageConfig(Constants.MODID + ":home_tree/starts", 10));
```

Comment:
We don't want this in all biomes do we? Shouldn't it just be jungle?

Reply:
Right right, forgot to pull this out.